release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -705,6 +705,11 @@ In {product-title} 4.12, the External DNS Operator modifies the format of the Ex
 
 The upstream version of `ExternalDNS` for {product-title} 4.12 is v0.13.1.
 
+[id="ingressClassName-required_{context}"]
+==== The ingressClassName field is required for each ingress object
```

Comment:
If you avoid "ingressClassName" in here entirely without hurting scanning, that might be nice, but NBD.

Reply:
Is this the right place for this item? It's mixed in with new features at the moment.

You might want to verify that a change to the release notes for this is necessary at all.